pkg/util/utils_test.go

```diff
@@ -88,45 +85,6 @@ func TestRetryNotRetriableError(t *testing.T) {
 	}
 }
 
-type getTestArgs struct {
-	input         string
-	expected      string
-	expectedError bool
-}
-
-var testSHAString = "test"
-
-func TestParseSHAFromURL(t *testing.T) {
```

Comment:
do we no longer need this test?

Reply:
I didn't find the test valuable, as the function shouldn't have been public in the first place. I took a second look, and discovered that this function is entirely unused, so I've opted to remove it entirely. Thanks for questioning it!